### ISSUE TITLE
When using addPostFrameCallback, exception occurs

### ISSUE THREAD

**Issue by kmccmk9** [simformsolutions/flutter_showcaseview#14]
Hi, we've encountered another interesting issue. When we have a startShowcase call within the addPostFrameCallback function within the build method, we get this stack trace and app crash. NOTE: if we start the app with the transition screen set as the home screen of the MaterialApp, we do not get any errors and everything works as expected. 

It seems to be hinting that the state that startShowcase relies on is null, but I'm not sure why that would be.
```

E/flutter (22985): [ERROR:flutter/lib/ui/ui_dart_state.cc(148)] Unhandled Exception: NoSuchMethodError: The method 'startShowCase' was called on null.
E/flutter (22985): Receiver: null
E/flutter (22985): Tried calling: startShowCase(Instance(length:1) of '_GrowableList')
E/flutter (22985): #0      Object.noSuchMethod  (dart:core-patch/object_patch.dart:50:5)
E/flutter (22985): #1      ShowCaseWidget.startShowCase 
package:showcaseview/showcase_widget.dart:22
```

**Comment by simformsolutions** [comment]
@kmccmk9  we're working on it, update you here once it done.

**Comment by kmccmk9** [comment]
Update: I discovered in the previous version if the showcase widget was wrapping a FloatingActionButton, everything worked normally. In the current version, it causes this stack trace or sometimes it switches to `activeWidgetIds was called on null`. Removing the showcase widget around the floating action button removes the crash.

**Comment by SanketKachhela** [comment]
It seems we'll need to use builder as second screen unable to get showcase widget state hence it throws exception.

So can you please try once by wrapping builder on ShowCaseWidget and user builderContext while start startShowCase

> ShowCaseWidget.startShowCase(
>             context, [_one, _two, _three, _four, _five]));

It seems similar issue while user getting to show snackbar with scaffold.

**Comment by nihalreddy33** [comment]
@SanketKachhela any ETA by when this issue will be resolved?

**Comment by SanketKachhela** [comment]
@nihalreddy33 about to release it by today

**Comment by SanketKachhela** [comment]
@kmccmk9 @nihalreddy33  issue has been fixed in version 0.1.3 . we've wrapped build to fix context issue so please check updated example codebase to get more idea.
